README.md

```diff
@@ -34,4 +34,26 @@ In a props file:
 
 ## Rules
 * [**Proj0001** .NET project file could not be located](rules/Proj0001.md)
-* [**Proj1001** "Use analyzers for packages](rules/Proj1001.md)
+* [**Proj0003** Define usings explicit](rules/Proj0003.md)
```

Comment:
There is no Proj0002?

Reply:
Not yet. It reserved for another rule I have in mind.